Frameworks/UIKit/UIViewController.mm

```diff
@@ -715,7 +715,7 @@ - (NSString*)nibName {
         NSString* ret = nil;
         NSString* nibPath;
 
-        char* ourClass = IwStrDup(object_getClassName(self));
+        char* ourClass = _strdup(object_getClassName(self));
```

Comment:
>_strdup [](start = 25, length = 7)

Unrelated to change, but it's unclear why it's necessary to duplicate this string at all.